src/components/panes/CallAssignmentPane.jsx

```diff
@@ -161,10 +162,7 @@ export default class CallAssignmentPane extends PaneBase {
                         onClick={ this.onClickEditTarget.bind(this) }/>
                     <Link msgId="panes.callAssignment.target.editGoalLink"
                         onClick={ this.onClickEditGoal.bind(this) }/>
-                    <div className="CallAssignmentPane-progress">
-                        <div style={{ width: progressSum + '%' }} 
-                            className="CallAssignmentPane-progressContent"/>
-                    </div>
+                    <ProgressBar progressSum={ progressSum }/>
```

Comment:
After having renamed the property to `progress` as discussed, this needs to updated.